wallet-new/src/Cardano/Wallet/API/V1/Accounts.hs

```diff
@@ -31,4 +31,8 @@ type API
           :> Summary "Update an Account for the given Wallet."
           :> ReqBody '[ValidJSON] (Update Account)
           :> Put '[ValidJSON] (WalletResponse Account)
+    :<|> "external-wallets" :> CaptureWalletId :> "accounts"
+          :> Summary "Creates a new Account for the given external Wallet."
+          :> ReqBody '[ValidJSON] (New Account)
+          :> Post '[ValidJSON] (WalletResponse Account)
```

Comment:
Currently it is assumed that external wallet has only one account. But in the future it may change...